examples/configs/sensors/FLIR_rgb_1.yml

```diff
@@ -1,7 +1,7 @@
 sensor_id: FLIR_rgb_1
-temporal_res: 195.0
-spatial_res_x: 548.0
-spatial_res_y: 800.0
+fps: 195.0
```

Comment:
This is fine, but could call it sample_rate for consistency with other types of devices.

Reply:
I used the names used in the sensor/device classes. I can change if you like

Reply:
I think it makes sense to change throughout for uniformity.

Reply:
I made the name changes you suggested. I think they're mostly consistent, but was conservative. W/o looking at the documentation for each device it's hard to be sure what's correct since sometimes we use the same name for different things and sometimes different names for the same things.